src/mavis/tools/vcf.py

```diff
@@ -204,6 +202,11 @@ def convert_record(record: VcfRecordType) -> List[Dict]:
                     COLUMNS.break2_position_end: end + info.get('CIEND', (0, 0))[1],
                 }
             )
+        if std_row['break1_position_end'] == 0 and std_row['break1_position_start'] == 1:
```

Comment:
I think the second half of this if is superfluous, we don't actually care what the start is, if the end is 0 it will be invalid no matter what. In fact I might even go so far as to just do this in a loop

```python
for pos in ['break1_position_start', 'break1_position_end', 'break2_position_start', 'break2_position_end']
  if std_row.get(pos) == 0:
    std_row[pos] = 1
```